browser.js

```diff
@@ -44,10 +56,31 @@ function pino (opts) {
     return new Child(this)
   }
   logger.levels = pino.levels
+  return !opts.browser.asObject ? logger : asObject(logger, levels)
+}
+
+function asObject (logger, levels) {
+  var k
+  for (var i in levels) {
```

Comment:
`levels` is an array in here.
It looks a bit strange that you're using a normal for loop everywhere else except here.